src/js/users/components/ManageUsers.tsx

```diff
@@ -11,6 +13,7 @@ import { UsersList } from "./UsersList";
  * @returns The user management view
  */
 export function ManageUsers() {
+    const history = useHistory<{ createUser: boolean }>();
```

Comment:
I think this still makes more sense as part of `CreateUser`. The strange desync issue experienced in testing seems worth confronting now, imo as we will likely have to deal with this down the road. If you haven't already, take some time to confirm the source of the problem and hunt for some other solutions. From my reading we should be using `MemoryRouter` more than we are in testing so we need to figure this out anyway.